<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой - Игра</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(
    x1: 0, y1: 0,
    x2: 1, y2: 1,
    stop: 0 #a0e9ff,
    stop: 1 #00bcd4
);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="player1Grid"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="player2Grid"/>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-weight: bold;
color: #2C3E50;
padding: 10px;
border-bottom: 2px solid #2980B9;
background-color: #ECF0F1;
qproperty-alignment: 'AlignCenter';
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimer">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2c3e50;
color: #ecf0f1;
font-size: 22px;
padding: 4px 10px;
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Морской Бой - Игра</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
